cookbooks/civicrm/recipes/default.rb

```diff
@@ -91,6 +91,18 @@
   site "supporting.openstreetmap.org"
 end
 
+wordpress_theme "morden" do
+  theme "morden"
```

Comment:
You don't need to repeat the theme name if it's the same as the resource name - it's the name property so is defaulted automatically.